src/shared/components/BottomSheet/BottomSheet.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import { createPortal } from 'react-dom';
+import { useEffectOnce } from 'react-use';
+
+import * as S from './BottomSheet.styles';
+import { BottomSheetProps } from './BottomSheet.types';
+
+const Component = ({ title, children }: BottomSheetProps) => {
+  return (
+    <S.Wrapper>
+      <S.Title>{title}</S.Title>
+      <S.Content>{children}</S.Content>
+    </S.Wrapper>
+  );
+};
+
+const BottomSheet = (props: BottomSheetProps) => {
+  const [element, setElement] = useState<HTMLElement | null>(null);
+
+  useEffectOnce(() => {
+    setElement(document.getElementById('bottom-sheet'));
+  });
```

Comment:
제목만 봐서는 한번만 실행된다는 소리 같은데,
아래 코드와 어떤 점이 다른건가용??
```suggestion
  useEffect(() => {
    setElement(document.getElementById('bottom-sheet'));
  }, []);
```

Reply:
다른점은 사실 없습니다! ㅋㅋㅋ
그냥 깔끔해보이고 눈에 확 띄어서 저렇게 사용했어용